motorApp/MclennanSrc/devPM304.cc

```diff
@@ -350,11 +380,27 @@ STATIC RTN_STATUS PM304_build_trans(motor_cmnd command, double *parms, struct mo
         sprintf(buff, "%dAB;", axis);
         break;
 
+   /* limits may or may not be enforced depending on last SL command 
+      hardware will not let you set a low limit >= high limit so order of setting
+      may be important. Need to look more closely at motor record */ 
     case SET_HIGH_LIMIT:
+        if (false) {
+            sprintf(buff, "%dUL%ld;", axis, ival);
```

Comment:
surely this will never happen?

Reply:
indeed - it is a bit of a todo/work in progress. It would be nice to set the limits, but reading back current values is difficult so there is a danger of setting invalid ones. The hardware rejects setting max < min so you have to be careful changing limits and make sure both are actioned. but i am wondering if just doing it twice catches that.